Formula/apache-flink.rb

```diff
@@ -1,9 +1,9 @@
 class ApacheFlink < Formula
   desc "Scalable batch and stream data processing"
   homepage "https://flink.apache.org/"
-  url "https://www.apache.org/dyn/closer.lua?path=flink/flink-1.2.0/flink-1.2.0-bin-hadoop27-scala_2.11.tgz"
-  version "1.2.0"
-  sha256 "ccb37938fef54236252a3e895cdca63d881c6138515109b4ad4da2a0fb9b8ac8"
+  url "https://www.apache.org/dyn/closer.lua?path=flink/flink-1.2.1/flink-1.2.1-bin-hadoop27-scala_2.11.tgz"
+  version "1.2.1"
+  sha256 "903dfdb0141c141d120a28653666cf9cbd8d86cd3bcc7540a058fae03aeccd90"
   head "https://github.com/apache/flink.git"
 
   bottle :unneeded
```

Comment:
After the `bottle` line, this formula needs a `depends_on :java` line (if any Java version is fine) or `depends_on :java => "1.7+"` (if there is a minimum version of, for example, Java 7)